addons/im_livechat/static/src/new/core/store_service.js

```diff
@@ -0,0 +1,21 @@
+/** @odoo-module */
+
+import { Store } from "@mail/new/core/store_service";
+import { _t } from "@web/core/l10n/translation";
+import { patch } from "@web/core/utils/patch";
+
+patch(Store.prototype, "im_livechat", {
+    setup(env) {
+        this._super(env);
+        this.discuss.livechat = {
+            extraClass: "o-mail-category-livechat",
+            id: "livechat",
+            name: _t("Livechat"),
+            isOpen: true,
+            canView: false,
+            canAdd: false,
+            addTitle: _t("Start a conversation"),
```

Comment:
this title looks wrong for a livechat as it's not possible for the operator to start a conversation

Reply:
Remove. `canAdd` is false, there is no reason to specify `addTitle`.